src/core_plugins/kibana/public/dashboard/dashboard.html

```diff
@@ -2,11 +2,13 @@
   <div class="fullScreenModePlaceholder">
     <div
       ng-if="fullScreenMode"
-      class="kuiButton exitFullScreenMode"
+      class="exitFullScreenMode"
       ng-click="exitFullScreenMode()"
+      tabindex="0"
```

Comment:
Can we remove `tabindex` and use `kbn-accessible-click` to make this keyboard accessible? Or even better, can we use a `button` element and the tooltip directive instead of the custom "Exit fullscreen" text? When I looked at the animation, it wasn't clear to me that it was just a label and not a button itself.